kangaroo.config.ts

```diff
@@ -1,15 +1,18 @@
 import { defineConfig } from './src/main/defineConfig';
 
 export default defineConfig({
-  appId: 'org.holochain.kangaroo-electron',
-  productName: 'Holochain Kangaroo Electron',
+  appId: 'org.holochain.kangaroo-electron.configtest',
+  productName: 'Holochain Kangaroo Electron (configtest)',
```

Comment:
```suggestion
  productName: 'Holochain Kangaroo Electron (Test Config)',
```

Reply:
oops, I keep accidentally committing these appId name changes. They're only changed for testing it locally because we're throwing an error if the default values are kept (otherwise people can end up accidentally deploying different apps under that same default appId).

Reply:
should be fixed now.

Reply:
b48a9097ef735dfa426221affef4dd7e70354c71